examples/arrays/arrays.description

```diff
@@ -1,6 +1,5 @@
 // Arrays are mutable lists of values of dynamic length in which each member in the list is
 // of the same type.
-// A fixed length array can be created by specifying the length of the array at the time of declaration.
-//
+// A fixed length array can be created by specifying the length of the array at the time of declaration.</br>
```

Comment:
Are we trying to leave a line in-between?

Reply:
Yes.